Sources/SWBTaskExecution/DynamicTaskSpecs/ClangModuleDependencyGraph.swift

```diff
@@ -153,33 +158,36 @@ package final class ClangModuleDependencyGraph {
             self.modules = moduleDependencies
             self.workingDirectory = workingDirectory
             self.commands = [command]
+            self.scanningCommandLine = scanningCommandLine
             self.transitiveIncludeTreeIDs = transitiveIncludeTreeIDs
             self.transitiveCompileCommandCacheKeys = transitiveCompileCommandCacheKeys
             self.usesSerializedDiagnostics = usesSerializedDiagnostics
         }
 
         package func serialize<T>(to serializer: T) where T : Serializer {
-            serializer.serializeAggregate(9) {
+            serializer.serializeAggregate(10) {
                 serializer.serialize(kind)
                 serializer.serialize(files)
                 serializer.serialize(includeTreeID)
                 serializer.serialize(modules)
                 serializer.serialize(workingDirectory)
                 serializer.serialize(commands)
+                serializer.serialize(scanningCommandLine)
```

Comment:
We should try to measure the impact of serializing an extra command line here, it should be shorter than the cc1 command but we do want to keep this data small...

Reply:
Is the impact in the consumed storage space? Just to know what to measure.

Reply:
I've tested the storage space changes on [Sparkle](https://github.com/sparkle-project/Sparkle) and here are my findings. The build directory has grown by 2.3MB (2,441,200 bytes) which is 0.5% of the build directory size (growth from 450,234,890 bytes to 452,676,090). Most of the growth (1.8MB) is in ExplicitPrecompiledModules where each .scan file (out of 647) has grown by 2.5-3KB. For the record, the smallest .pcm in my testing is 20KB, average is 366KB. I wasn't using short names or shallow directory structure, so the measurements should be fairly representative.

Is it a sufficient measurement or would you like to see more data?